.travis.yml

```diff
@@ -1,18 +1,26 @@
-language: php
-
 sudo: false
 
+language: php
+
 cache:
-  directory:
+  directories:
     - $HOME/.composer/cache
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7
-  - hhvm
+    - 7
+    - hhvm
+
+install:
+    - composer self-update
+    - composer --prefer-source install
```

Comment:
@FabioBatSilva 

If you actually want to cache dependencies installed with composer between builds, they need to be installed with `--prefer-dist`.